pkg/graph/schema.graphql

```diff
@@ -1018,12 +1018,27 @@ type SystemIntake {
   contractName: String
   relationType: SystemIntakeRelationType # TODO: NOT IMPLEMENTED
 
+  """
+  Linked systems
+  """
+  systems: [SystemIntakeSystem!]!
```

Comment:
I think it'd be useful to have a resolver that allows fetching the [actual CEDAR System type](https://github.com/CMSgov/easi-app/blob/3289a1307f8e40bc76f3b92e2bbba3b5cb7aac89/pkg/graph/schema.graphql#L89-L104). Perhaps adding a field to the `SystemIntakeSystem` type called `CEDARSystem` that fetches it?

Reply:
It's also entirely possible (and reasonable) to just not use this custom `SystemIntakeSystem` type at all, and just expose the `CedarSystem` type right here